src/js/view/controls/components/menu/settings-menu.js

```diff
@@ -342,6 +342,9 @@ class SettingsMenu extends Menu {
                 }
                 focusEl = gearButton;
                 break;
+            case 'Esc':
+                focusEl = gearButton;
+                break;
```

Comment:
This is a great place to use [switch statement fall-through](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch#can_i_put_a_default_between_cases):

```suggestion
                /* falls through */
            case 'Esc':
                focusEl = gearButton;
                break;
```

The `/* falls through */` comment lets the linter know you mean to leave out `break;` so that in the `Tab` base, execution will continue through the `Esc` case. This lets us not have to repeat  `focusEl = gearButton` twice, since it's used at the end of `Tab` and and alone in `Esc`.

Reply:
Scroll up in the code 6 lines @robwalch, we don't want to treat it like tab.

Reply:
Ah this is actually pretty clever, took me a second to see.

Reply:
ahh got it! thanks for the suggestion. i'll go ahead and make those changes

Reply:
waiting to get re-added to the team_players org before I can push up the new commit